action.yml

```diff
@@ -251,16 +251,54 @@ runs:
         [[ "$GIT_PR" =~ ^[0-9]+$ ]] && GIT_PR_MSG="<$GIT_PR_URL|$GIT_COMMIT_MSG>" || GIT_PR_MSG="$GIT_COMMIT_MSG"
         echo "GIT_PR_MSG=$GIT_PR_MSG" >> $GITHUB_ENV
 
-    - name: Report - When Success [on master]
+    - name: Report - success [on master]
       if: success() && github.ref == 'refs/heads/master'
-      uses: Ilshidur/action-slack@2.1.0
+      uses: slackapi/slack-github-action@v1.23.0
       env:
-        SLACK_WEBHOOK: ${{ inputs.up-slack-webhook-url }}
-        SLACK_CUSTOM_PAYLOAD: '{"channel": "1up-releases", "attachments":[{"color": "#2eb886", "blocks":[{"type":"section","text":{"type":"mrkdwn","text": "{{ GIT_ACTIONS_URL }} released\n{{ GIT_PR_MSG }}"}}]}]}'
-
-    - name: Report - When Failure [on master]
+        SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        SLACK_WEBHOOK_URL: ${{ inputs.up-slack-webhook-url }}
+      with:
+        channel-id: "1up-releases"
```

Comment:
why do we need to define this (should this have been part of webhook?)

Reply:
> 

Since you often come up with this topic and Cat continues to reply likewise, we might simply _talk_ about our commit messages at some point? What do you think?

Reply:
You can put the channel id in the payload too but since there is a specific param for it, why not use it. That's what I thought :)

Reply:
I mean we create a webhook for a specific channel, like if you pass here another channel I'd expect the message to not be sent, or be sent to the channel which webhook is attached to

Reply:
I haven't tried but the old code also specified the channel. What would you like to change here?

Reply:
Tried. The channel is fixed, cannot override. Should I remove the channel param?

Reply:
I leave this to you - I'm ok having it or not having it. I'm slightly inclined to not have it because we are passing a webhook and it is a parameter - so ideally channel also should be a parameter.

But, I'm not against leaving it as it is now